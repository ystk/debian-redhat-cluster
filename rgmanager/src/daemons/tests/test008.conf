<?xml version="1.0"?>
<!-- When comapred with test7,
     there should be no NEEDSTART flags in any of the output trees, because
     although we have now defined "mount1", we have not assigned it
     to any resource groups.  There SHOULD be a NEEDSTART in the resource
     list, just not any of the trees. -->
<cluster>
<rm>
	<resources>
    		<service name="test1"/>
		<script name="initscript" file="/etc/init.d/sshd"/>
		<ip address="192.168.1.3" monitor_link="yes"/>
    		<fs fstype="ext3" name="mount1" mountpoint="/mnt/cluster" device="/dev/sdb8"/>
	</resources>
	<service ref="test1">
		<ip ref="192.168.1.3"/>
		<script ref="initscript"/>
	</service>
</rm>
</cluster>
