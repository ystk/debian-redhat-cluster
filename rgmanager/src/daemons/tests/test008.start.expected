Starting test1...
[start] service:test1
[start] ip:192.168.1.3
[start] script:initscript
Start of test1 complete
